vendor.mod

```diff
@@ -1,6 +1,7 @@
 // 'vendor.mod' enables use of 'go mod vendor' to managed 'vendor/' directory.
-// There is no 'go.mod' file, as that would imply opting in for all the rules
-// around SemVer, which this repo cannot abide by as it uses CalVer.
+// There is no 'go.mod' file, as the project has not yet completed the migration
+// to a Go module. This project should be built in GOPATH mode, and not module mode,
+// until the migration is complete and this file becomes 'go.mod.'
```

Comment:
Can we have a link to an issue to track the blockers for the migration?

Reply:
I've been bugging @thaJeztah about making one; I'll link it in these places once we have one :+1: